python/src/Function.i

```diff
@@ -435,4 +435,6 @@ class PythonFunction(Function):
             instance.isVariableLinear = variablesLinearity
                           
         return Function(instance)
+
+Function._repr_html_ = Function.__repr_html__
```

Comment:
oh, so the right python name is actually _repr_html_
it should be renamed in the cxx side so this alias can be dropped then (else it wont be available for other objects)

Reply:
I thought of this way, but it breaks the consistency of the C++ API naming:
```cpp
// Option 1 : rename in C++ so that Python can recognize the naming
virtual String __repr__() const;
virtual String _repr_html_() const;  // or __repr_html__ ?
virtual String __str__(const String & offset = "") const;
```
In other words, the C++ shows doubled underscores, except for the `html` repr. The advantage of this naming is the corresponding Python classes are automatically pretty-printed, without any need for the `.i` trick.

The option 2 is as currently implemented in the PR. I agree with you: this means that only classes that have the `_repr_html_` trick in their `.i` will get pretty-printed. One drawback is that we choose the C++ name `_ repr_html_` not based on an internal C++ standard way of programming, but based on the naming imposed on the Python side.

I did not find any C++ standard on this topic: this way of programming seems to be Python-specific.

I do not known if you think of these details when posting your comment. Which option should I pick finally?

Reply:
Use `_repr_html_`

Reply:
Ok. Fixed at 1b4003f67c39301cfeb121c6446786b08b032249.